packages/driver-interface/src/services/bot.ts

```diff
@@ -7,39 +6,34 @@ import * as messaging from './messaging'
 import { IncomingMessage, Message } from 'telegraf/typings/telegram-types'
 import { TelegrafContext } from 'telegraf/typings/context'
 
+export const driverIsLoggedIn = async (vehicleId) => {
+  const vehicle = await cache.getVehicle(vehicleId)
+  return !!vehicle.telegramId
+}
+
 export const onLogin = async (
-  vehicleId: string,
+  phoneNumber: string,
   ctx: TelegrafContext
 ): Promise<Message | void> => {
+  const vehicleId = await cache.getVehicleIdByPhoneNumber(
+    phoneNumber.replace('46', '0')
```

Comment:
Sometimes numer comes as +46, sometimes as just 46.

Reply:
My number is registered as +46 in the telegram app but it always sends 46. How can I reproduce what you're saying?

Reply:
When I did this in telegram from the computer. Don't know if that is the only case, but at least, we cannot be sure it is just without + sign